src/controller/AutoCommissioner.h

```diff
@@ -28,15 +28,14 @@ class DeviceCommissioner;
 class AutoCommissioner : public CommissioningDelegate
 {
 public:
-    AutoCommissioner(DeviceCommissioner * commissioner) : mCommissioner(commissioner) {}
-    ~AutoCommissioner();
-    CHIP_ERROR SetCommissioningParameters(const CommissioningParameters & params);
-    void SetOperationalCredentialsDelegate(OperationalCredentialsDelegate * operationalCredentialsDelegate);
+    AutoCommissioner() {}
+    virtual ~AutoCommissioner();
+    virtual CHIP_ERROR SetCommissioningParameters(const CommissioningParameters & params) override;
+    virtual void SetOperationalCredentialsDelegate(OperationalCredentialsDelegate * operationalCredentialsDelegate) override;
```

Comment:
```suggestion
    CHIP_ERROR SetCommissioningParameters(const CommissioningParameters & params) override;
    void SetOperationalCredentialsDelegate(OperationalCredentialsDelegate * operationalCredentialsDelegate) override;
```
`override` implies `virtual`, so it's just noise to have both... Same for the other APIs here.